frontend/packages/core/src/AppLayout/stories/user-information.stories.tsx

```diff
@@ -1,23 +1,28 @@
 import * as React from "react";
-import { Grid } from "@material-ui/core";
+import styled from "@emotion/styled";
+import { Grid as MuiGrid } from "@material-ui/core";
 import type { Meta } from "@storybook/react";
 
+import type { UserInformationProps } from "../user";
 import { UserInformation } from "../user";
 
 export default {
   title: "Core/AppLayout/User Information",
   component: UserInformation,
-  parameters: {
-    backgrounds: {
-      default: "header blue",
-      values: [{ name: "header blue", value: "#131C5F" }],
-    },
-  },
 } as Meta;
 
-const Template = () => (
+const Grid = styled(MuiGrid)({
+  height: "64px",
+  backgroundColor: "#131C5F",
+});
+
+const Template = (props: UserInformationProps) => (
   <Grid container alignItems="center" justify="center">
-    <UserInformation />
+    <UserInformation {...props} />
   </Grid>
 );
 export const Primary = Template.bind({});
+Primary.args = {
+  data: [{ value: "Dashboard" }, { value: "Settings" }],
+  user: "fooBar@gmail.com",
```

Comment:
super nit: use example domain

```suggestion
  user: "fooBar@example.com",
```